provisioning/service/samples/createCertificate.js

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
>Copyright [](start = 3, length = 9)

Rats!  I didn't see this PR until just now after I submitted a PR that included a similar tool.  See [https://github.com/Azure/azure-iot-sdk-node/pull/198](https://github.com/Azure/azure-iot-sdk-node/pull/198)

Reply:
yeah at that point we probably need to merge these tools.

And then consider merging this into iothub-explorer.

Reply:
Disagree on putting this into iothub-explorer.  This is just a wrapper around PEM and has no actual connection to the hub code.  Also, these certs are test-only.  We don't want to get into the business of telling people how to make production certs.  They should have their own process for that.  

---
In reply to: [161137256](https://github.com/Azure/azure-iot-sdk-node/pull/196#discussion_r161137256) [](ancestors = 161137256)

Reply:
fair enough - thanks for chiming in :) 

---
In reply to: [161297082](https://github.com/Azure/azure-iot-sdk-node/pull/196#discussion_r161297082) [](ancestors = 161297082,161137256)